### ISSUE TITLE
training speed ...

### ISSUE THREAD

**Issue by Gavin666Github** [xw-hu/Mask-ShadowGAN#3]
I use GTX 1080 TI to train the model ,but it's too slow,i wonder how long did you train the code for 100 epochs?

**Comment by XSLXANDY** [comment]
the same question...   Hey, Gavin666Github, how long did you train the code for 1 epoch?

**Comment by naoto0804** [comment]
About a 0.33 hour per epoch on Tesla V100.

**Comment by xw-hu** [comment]
It requires CPU to process the queue of shadow masks (QueueMask). You can reduce the length of the queue of shadow masks (QueueMask(dataloader.__len__()/4)) to make it faster, and the performance is similar.
